app/src/main/java/com/cornellappdev/score/components/SportCard.kt

```diff
@@ -31,16 +36,39 @@ fun SportCard(
     location: String,
     genderIcon: Painter,
     sportIcon: Painter,
+    topCornerRound: Boolean,
     modifier: Modifier = Modifier
 ) {
+    val cardShape = if (topCornerRound) {
+        RoundedCornerShape(16.dp) // Rounded all
+    } else {
+        RoundedCornerShape(topStart = 0.dp, topEnd = 0.dp) // square top corners
+    }
```

Comment:
I can't say I exactly follow this logic. So if `topCornerRound` is true, then you make all corners round, and otherwise, you make all of them square? Is this second `RoundedCornerShape(topStart = 0.dp, topEnd = 0.dp)` call making anything rounded at all?

Reply:
The default shape for the Card is round corners. To make the top corners not square for the SportCard section in the Upcoming Game Card, I'm using a RoundedCornerShape with topStart and topEnd set to 0.dp.